sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -23,7 +23,7 @@ object DottyPlugin extends AutoPlugin {
     val isDotty = settingKey[Boolean]("Is this project compiled with Dotty?")
     val isDottyJS = settingKey[Boolean]("Is this project compiled with Dotty and Scala.js?")
 
-    val useScala3doc = settingKey[Boolean]("Use Scala3doc as the documentation tool")
+    val useScaladoc = settingKey[Boolean]("Use scaladoc as the documentation tool")
```

Comment:
Does the new name really make sense here? Perhaps we should call this key `useNewScaladoc`?

Reply:
Next step would be removing scala3-doc and this setting as well.